Directory.Build.props

```diff
@@ -172,6 +172,10 @@
       freebsd-x64
     </SupportedRuntimeIdentifiers>
 
+    <!-- Playwright provides binaries for Windows (x86 and x64), macOS (x64) and Linux (x64, non musl). We can't use it on other architectures. -->
+    <IsPlaywrightAvailable Condition="'$(TargetOsName)' == 'linux-musl' OR ('$(TargetArchitecture)' != 'x86' AND '$(TargetArchitecture)' != 'x64')">false</IsPlaywrightAvailable>
+    <IsPlaywrightAvailable Condition="'$(IsPlaywrightAvailable)' == ''">true</IsPlaywrightAvailable>
```

Comment:
Just cross-linking given the Playwright changes: https://github.com/dotnet/aspnetcore/pull/38414

Reply:
Thanks! Looking at https://nuget.info/packages/Microsoft.Playwright/1.17.0, it looks like it has even fewer os/arch combinations: Linux, macOS, and Windows all x64 only.

Reply:
/fyi if I'm correct and we only use Playwright in our BlazorTemplates.Tests, then we currently use Playwright only on macOS and Windows in the CI. Testing on Linux x64 (or at least Ubuntu 18.04 / 20.04 x64) would be nice to have but I don't know if that even works locally (@TanayParikh ❔)

Reply:
> Testing on Linux x64 (or at least Ubuntu 18.04 / 20.04 x64) would be nice to have but I don't know if that even works locally (@TanayParikh ❔)

I'm currently upgrading our Playwright usage from `v0.192.0` to `v1.16.1` via https://github.com/dotnet/aspnetcore/pull/38414. This involves a bunch of breaking changes, so currently the tests don't really work anywhere. Once that's all up and running again, I'm anticipating there shouldn't be too big of an issue running the tests on linux (along with Windows and macOS).

Side note; it looks like `v1.17.0` just got released so will look into updating my PR to use that version.